### DISCUSSION TITLE
Double-click to edit

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14008] [Feature improvement]
Older brother have "double click to edit record" in list view.  It little java thing to do, but improving interaction with system a lot, 
Modern web apps usually following this pattern. As YF design goes pretty contemporary it would be advantage to add this functionality to the box

**Comment by JohnMaverick** [comment]
Most of the call centers working on list view only. This is function is crucial if yetiforce want to fight with competition
